apps/prairielearn/src/pages/authCallbackLti/authCallbackLti.js

```diff
@@ -79,8 +78,7 @@ router.post('/', function (req, res, next) {
       // https://oauth.net/core/1.0/#nonce
       var nonceReused = false;
       var nonceKey = 'authCallbackLti:' + parameters.oauth_timestamp + ':' + parameters.oauth_nonce;
-      util.callbackify(cache.get)(nonceKey, (err, val) => {
-        if (ERR(err, next)) return;
+      cache.get(nonceKey).then((val) => {
```

Comment:
This doesn't do proper error handling and is just generally broken (it was before you touched it). I opened https://github.com/PrairieLearn/PrairieLearn/pull/9384 to rewrite this whole file so we can use normal `async`/`await` here.

Reply:
That PR is merged, you should be good to get this branch up to date with master!

Reply:
Great! Updated this branch in 156b26e.